packages/apollo-language-server/src/workspace.ts

```diff
@@ -123,20 +124,26 @@ export class GraphQLWorkspace {
     const projectConfigs = Array.from(apolloConfigFolders).map(configFolder =>
       loadConfig({ configPath: configFolder, requireConfig: true })
         .then(config => {
-          foundConfigs.push(config);
-          const projectsForConfig = config.projects.map(projectConfig =>
-            this.createProject({ config, folder })
-          );
-
-          const existingProjects =
-            this.projectsByFolderUri.get(folder.uri) || [];
-
-          this.projectsByFolderUri.set(folder.uri, [
-            ...existingProjects,
-            ...projectsForConfig
-          ]);
+          if (config) {
```

Comment:
Had to modify this file to be able to accept null configs. Since the new pattern would be to return empty responses and log errors, we'll need to do this a few places.